applications/asset_tracker_v2/overlay-agps.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Set nRF Cloud as source for handling incoming A-GPS and P-GPS data.
+# This enables the application to use nRF Cloud specific APIs to process
+# incoming A-GPS and P-GPS data that has indirectly been fetched from nRF Cloud
+# on the cloud side also when nRF Cloud is not used as the cloud backend.
+CONFIG_AGPS=y
+CONFIG_AGPS_SRC_NRF_CLOUD=y
+CONFIG_NRF_CLOUD_AGPS=y
+
+# A-GPS using SUPL server.
+# Due to licensing, the SUPL library must be manually downloaded and configured
+# according to the instructions found in the SUPL client library documentation in NCS.
+# CONFIG_AGPS_SRC_SUPL=y
```

Comment:
`CONFIG_AGPS_SRC_SUPL` was removed in #5882

Reply:
The A-GPS configuration is for the A-GPS library, which is not used anymore. Please check the current configuration and update accordingly. Enabling `CONFIG_NRF_CLOUD_AGPS` should be enough. The comment has been updated a bit as well.

@simensrostad But, is it intentional that A-GPS is not used by default like before? Now the application has to be built with the A-GPS overlay to enable A-GPS support. Is that what we want?

Reply:
Thanks! @tokangas Good point. We should have A-GPS enabled by default. I don't see any benefits of keeping `overlay-agps.conf` in that case. Ill remove it if there aren't any objections.